plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/PostgreServerRedshift.java

```diff
@@ -65,25 +65,90 @@ public PostgreServerRedshift(PostgreDataSource dataSource) {
     };
     
     public static String[] REDSHIFT_EXTRA_KEYWORDS = new String[]{
-        "AUTO",
-        "BACKUP",
-        "AZ64",
-        "CASE_SENSITIVE",
-        "CASE_INSENSITIVE",
-        "COMPOUND",
-        "INTERLEAVED",
-        "COPY",
-        "DATASHARE",
-        "DISTSTYLE",
-        "DISTKEY",
-        "EVEN",
-        "MODEL",
-        "OWNER",
-        "SORTKEY",
-        "TEMP",
-        "UNLOAD",
-        "VACUUM",
-        "YES"
+	    "AUTO",
+	    "BACKUP",
+	    "AZ64",
+	    "CASE_SENSITIVE",
+	    "CASE_INSENSITIVE",
+	    "COMPOUND",
+	    "INTERLEAVED",
+	    "COPY",
+	    "DATASHARE",
+	    "DISTSTYLE",
+	    "DISTKEY",
+	    "EVEN",
+	    "MODEL",
+	    "OWNER",
+	    "SORTKEY",
+	    "TEMP",
+	    "UNLOAD",
+	    "VACUUM",
+	    "YES",
+	    "AES128",
+	    "AES256",
+	    "ALLOWOVERWRITE",
+	    "BLANKSASNULL",
+	    "BYTEDICT",
+	    "BZIP2",
+	    "CREDENTIALS",
+	    "DEFLATE",
```

Comment:
We use 4 spaces indentation
```suggestion
	"AUTO",
	"BACKUP",
	"AZ64",
	"CASE_SENSITIVE",
	"CASE_INSENSITIVE",
	"COMPOUND",
	"INTERLEAVED",
	"COPY",
	"DATASHARE",
	"DISTSTYLE",
	"DISTKEY",
	"EVEN",
	"MODEL",
	"OWNER",
	"SORTKEY",
	"TEMP",
	"UNLOAD",
	"VACUUM",
	"YES",
	"AES128",
	"AES256",
	"ALLOWOVERWRITE",
	"BLANKSASNULL",
	"BYTEDICT",
	"BZIP2",
	"CREDENTIALS",
	"DEFLATE",
```